### PR TITLE
Add jupyterlab_server

### PR DESCRIPTION
**Pull request by blink1073**
This will be a requirement for the next version of JupyterLab.

### CODE REVIEW

File: recipes/jupyterlab-server/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+
+source:
+  fn: jupyterlab_server-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/j/jupyterlab_server/jupyterlab_server-{{ version }}.tar.gz
+  sha256: a5373cda6579ca4504b38068a0cb4eba3b72b032a0de44291062788abf6d0f43
+
+build:
+    number: 0
+    noarch: python
+    script: python -m pip install --no-deps  --ignore-installed .
```
Comment (marcelotrevisani) [inline recipes/jupyterlab-server/meta.yaml]: Please, change that to
```
    script: "{{ PYTHON }} -m pip install . --no-deps --ignore-installed --no-cache-dir -vvv"
```

File: recipes/jupyterlab-server/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+  license: BSD 3-Clause
+  license_file: LICENSE
+  summary: A set of server components for JupyterLab and JupyterLab like applications.
+
+extra:
+  recipe-maintainers:
+    - jasongrout
+    - blink1073
+    - afshin
+    - SylvainCorlay
```
Comment (marcelotrevisani) [inline recipes/jupyterlab-server/meta.yaml]: all of these people already agreed to be a maintainer of this recipe?

@blink1073 , @afshin , @SylvainCorlay

Comment (afshin) [reply recipes/jupyterlab-server/meta.yaml]: 👍 from me

Comment (SylvainCorlay) [reply recipes/jupyterlab-server/meta.yaml]: 👍

Comment (blink1073) [reply recipes/jupyterlab-server/meta.yaml]: cc @jasongrout

Comment (jasongrout) [reply recipes/jupyterlab-server/meta.yaml]: Yep!

**Comment by marcelotrevisani** [comment]
LGTM!

Thanks to pu this in ``conda-forge``!  :)

**Pull request merged**